presto-docs/src/main/sphinx/release/release-0.167-t.rst

```diff
@@ -0,0 +1,58 @@
+=================
+Release 0.167-t
+=================
+
+Presto 0.167-t is equivalent to Presto release 0.167, with some additional features and patches.
+
+**TIMESTAMP limitations**
+
+Presto supports a granularity of milliseconds for the ``TIMESTAMP`` datatype, while Hive
+supports microseconds.
+
+**TIMESTAMP semantic changes**
+
+The Teradata distribution of Presto fixes the semantics of the ``TIMESTAMP`` type to align with the SQL standard.
+
+Previously, the ``TIMESTAMP`` type described an instance in time in the Presto session's time zone.
+Now, Presto treats ``TIMESTAMPs`` as a set of the following fields representing wall time:
```

Comment:
The mixed case looks really weird.  Maybe capitalize the `s`. Or put it out of the quotes so it's formatted differently.  I'm not sure what your supposed to do in these cases.